packages/node/src/inference.ts

```diff
@@ -6,9 +6,14 @@ import { calculateProportionalSize } from './utils';
 import { Config, validateConfig } from './schema';
 import ndarray, { NdArray } from 'ndarray';
 import { loadAsBlob } from './resource';
+import pkg from '../package.json';
 
 async function initInference(config?: Config) {
   config = validateConfig(config);
+  // replace ${PACKAGE_NAME and ${PACKAGE_VERSION} in the path
+  config.publicPath = config.publicPath
```

Comment:
THis can be done directly in Zod with I thing `transform`

Reply:
So not extra outside to keep the concern of parsing etc to zod

Reply:
Makes sense, didn't know this method!